### DISCUSSION TITLE
Failure at initial step but always SSL CERTIFICATE_VERIFY_FAILED

### DISCUSSION THREAD

**Discussion by Jir8taiwan** [waydroid/waydroid#655] [Q&A]
Hi, Friends,

I have tried to google and waydroid document to know the reason, but not success in my limited knowledge.
Could I have any advise to resolve the problem about this experience?
It has been "sudo update-ca-certificates" and "export SSL_CERT_DIR=/etc/ssl/certs".
After copy prebuilt LIANGE rom to the storage folder (/usr/share/waydroid-extra/images/), it is still sending me the error reply for:
`sudo waydroid init -f`
**ERROR: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1056)>
See also: <https://github.com/waydroid>
Run 'waydroid log' for details.**

And I also try to use gui tool to have extenal download prebuilt rom.
It is still having similar error code in the diaglog message as shown:
**<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1056)>**

In the other side, I try to ouput a log file from "sudo waydroid log".
The log content is thinking problem from Python language poster, but I am not familiar with the program structure and how to fix the block line.
```
  File "/usr/lib/python3.7/urllib/request.py", line 1367, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/usr/lib/python3.7/urllib/request.py", line 1326, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1056)>
```

If someone can share detailly fixing solution, it may help me a lot to run the android emulator.
If above information is not enough, please tell me to refill them as full as I can.
Thanks a lot first.

1.
Capture:
![1671753156106](https://user-images.githubusercontent.com/70804014/209265419-07331740-1dcd-4e5d-9dc3-783535c37857.jpg)
2.
Log:
[waydroidlog221223.txt](https://github.com/waydroid/waydroid/files/10291822/waydroidlog221223.txt)
3.
cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
waydroid -V
1.3.4

**Comment by cniw** [comment]
don't forget install this package `sudo apt install ca-certificates`

if still doesn't works https://askubuntu.com/questions/1401379/certificate-verify-failed-error
